Runtime/Components/ConeRayAnglesEstimator/GuidedDataCollector.cs

```diff
@@ -0,0 +1,39 @@
+using UnityEngine;
+
+namespace ubco.ovilab.HPUI
+{
+    /// <summary>
+    /// Data is collected by specifying which segment all data should be assigned to.
+    /// </summary>
+    public class GuidedDataCollector : ConeRayDataCollectorBase
+    {
+        [SerializeField]
+        [Tooltip("Phalange that the interactor is currently being calibrated for")]
+        private HPUIInteractorConeRayAngleSegment targetSegment;
+
+        /// <summary>
+        /// Phalange that the interactor is currently being calibrated for
+        /// </summary>
+        public HPUIInteractorConeRayAngleSegment TargetSegment { get => targetSegment; set => targetSegment = value; }
+
+        /// <summary>
+        /// This will create a <see cref="ConeRayComputationDataRecord"/> for the
+        /// segment passed as a parameter.
+        /// </summary>
+        public void EndCalibrationForSegment(HPUIInteractorConeRayAngleSegment segment)
```

Comment:
If i decide to recollect the data for a phalange, will it overwrite or just add it to the pre-recorded Cone for that segment

Reply:
i like the overall structure, need to actually test it in the lab tomorrow before merging (and make sure the editor scripts are all working as expected)

Reply:
It would add another batch of data (ConeRayComputationDataRecord). You could override `DataRecorder`s get.

Reply:
Should make that prop virtual